### PR TITLE
Add unit tests and polish Javadocs with lambdas for PatternsCS. #24881

### PR DESCRIPTION
**Pull request by kun-song**

### CODE REVIEW

**Comment by johanandren** [comment]
Fixes #24881

**Comment by raboof** [comment]
OK TO TEST

**Comment by johanandren** [comment]
OK TO TEST

File: akka-actor-tests/src/test/java/akka/pattern/PatternsTest.java [hunk 1]
```diff
@@ -50,12 +53,34 @@ public Receive createReceive() {
...
+            public StopMessage(TestLatch latch, FiniteDuration duration) {
+                this.latch = latch;
+                this.duration = duration;
+            }
+        }
+
+        @Override
+        public Receive createReceive() {
+            return receiveBuilder()
+                    .match(StopMessage.class, message -> Await.ready(message.latch, message.duration))
```
Comment (johanandren) [inline akka-actor-tests/src/test/java/akka/pattern/PatternsTest.java]: No need to do this blocking, just reply and verify that a quick ping-pong works with the different asks

**Review by johanandren — CHANGES_REQUESTED** [review]

File: akka-actor-tests/src/test/java/akka/pattern/PatternsTest.java (hunk 1 above)
Comment (satansk) [reply akka-actor-tests/src/test/java/akka/pattern/PatternsTest.java]: Get it removed, and it certainly becomes more clear.

**Review by johanandren — APPROVED** [review]
Thanks, LGTM!

**Review by ktoso — APPROVED** [review]
LGTM, thanks!

**Pull request merged**